CorsixTH/Lua/hospital.lua

```diff
@@ -98,7 +98,16 @@ function Hospital:Hospital(world, avail_rooms, name)
   -- level; e.g. Easy: 0.4 / Difficult: 0.2)
   self.over_priced_threshold = 0.3
 
-  self.radiator_heat = 0.5
+  -- (int) Number of days until the next boiler or vomit wave disaster.
+  -- TODO: Implement the vomit wave.
+  self.disasterless_days = self:daysTillNextDisaster()
+
+  self.curr_setting = nil -- (float) Saved radiator heat when boiler has broken down.
```

Comment:
curr_setting seems a bit vague? For earthquake I put all the disaster related parameters in one table to separate them from the other world parameters - might be nice to do here too. https://github.com/CorsixTH/CorsixTH/blob/master/CorsixTH/Lua/world.lua#L100